### PR TITLE
Fix lint errors

### PR DESCRIPTION
**Pull request by smorimoto**
Ref: https://github.com/axiomhq/axiom-syslog-proxy/pull/61#issuecomment-2039595710

### CODE REVIEW

File: .golangci.yaml [hunk 1]
```diff
@@ -34,15 +42,9 @@ linters:
...
-linters-settings:
-  gofmt:
-    simplify: true
-  goimports:
-    local-prefixes: github.com/axiomhq/axiom-syslog-proxy
-  govet:
-    check-shadowing: true
-  nolintlint:
-    allow-unused: false
-    allow-leading-space: false
```
Comment (smorimoto) [inline .golangci.yaml:46]: - https://github.com/golangci/golangci-lint/pull/3002
- https://github.com/golangci/golangci-lint/pull/3072

**Review by lukasmalkmus — APPROVED** [review]
LGTM!

Really appreciate the detailed comments of yours!!!

**Comment by smorimoto** [comment]
My pleasure!

**Comment by lukasmalkmus** [comment]
@smorimoto Tagged `v0.7.1`.

**Comment by smorimoto** [comment]
Thanks a lot!

**Pull request merged**